dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilderTest.java

```diff
@@ -178,6 +183,33 @@ public void testGetDirectDependencies_respectExclusions() throws RepositoryExcep
     }
   }
 
+  @Test
+  public void testGetStaticLinkageCheckDependencyGraph_artifactProblems()
+      throws RepositoryException {
+    // In the full dependency tree of hibernate-core, xerces-impl:2.6.2 and xml-apis:2.6.2 are not
+    // available in Maven Central.
+    Artifact hibernateCore = new DefaultArtifact("org.hibernate:hibernate-core:jar:3.5.1-Final");
+
+    DependencyGraphResult result =
+        dependencyGraphBuilder.getStaticLinkageCheckDependencyGraph(
+            ImmutableList.of(hibernateCore));
+
+    ImmutableList<UnresolvableArtifactProblem> artifactProblems = result.getArtifactProblems();
+
+    Truth.assertThat(artifactProblems).hasSize(2);
+    UnresolvableArtifactProblem firstProblem = artifactProblems.get(0);
+    assertEquals("xerces:xerces-impl:jar:2.6.2", firstProblem.getArtifact().toString());
+
+    assertEquals(
```

Comment:
can we test this without converting it to a string?

Reply:
I can setup getter for that. Give me one moment.

Reply:
Maybe not worth it then.